### ISSUE TITLE
Solution for SSL and not

### ISSUE THREAD

**Issue by carmas123** [ClusterWS/ClusterWS#95]
### Submitting

- [ ] Bug
- [X ] Question
- [ ] Suggetion
- [ ] Other

### Details
Hi, is there a solution for listen on 443 and 80 port in same app?

**Comment by goriunov** [comment]
Unfortunately clusterws does not support that (if you are not running separately  2 servers one of them with 80 and another with 443) you can actually build redirect server which runs on port 80 and will automatically redirect to 443 or you can use nginx to proxy your 443 port to 80.

**Comment by carmas123** [comment]
I try to run 2 servers but it make multiple workers (if I set workers to 4 I got 8 workers)

**Comment by goriunov** [comment]
Yep that is right you will have divide workers between 2 servers, also you may need to change broker's port as you run 2 servers and default port will be taken by first server. (in my opinion it is better to use nginx and single server )
